alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportIndexMonitorAction.kt

```diff
@@ -711,7 +711,7 @@ class TransportIndexMonitorAction @Inject constructor(
                                 .execute(it)
                         }
                     }
-                    if (currentMonitor.owner == "alerting") {
+                    if (currentMonitor.deleteQueryIndexInEveryRun == false) {
```

Comment:
https://github.com/opensearch-project/alerting/blob/b280761831bc8accac799a316202a7e49899456e/alerting/src/main/kotlin/org/opensearch/alerting/util/DocLevelMonitorQueries.kt#L459
plz change it to error log